test/gtest/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ set(unittest_SOURCES
 
 if(BUILD_FUZZTEST)
   set(unittest_SOURCES ${unittest_SOURCES} type-domains.cpp)
+else()
+  set(unittest_SOURCES ${unittest_SOURCES} source-map.cpp)
```

Comment:
@tlively I've had to exclude this file from fuzztest builds, because it uses the gmock-matchers.h header (gmock is included with "standard" gtest but not with the one bundled with fuzztest). I tried including the gmock header along with the fuzztest gtest, but it seems to depend on some kind of internal gtest bit that the fuzztest gtest doesn't have. Do you know if fuzztest's bundled gtest is some sort of subset, or if they have any recommendations for use with gtest features they don't support themselves? Or maybe this is just WAI, and the intention is that fuzztests are logically separate enough from regular unit tests that it just makes sense to build them separately.

Reply:
No, I don't know the details of why fuzztest's bundled gtest is different from normal gtest, nor have I seen any recommendations for this. It's definitely not WAI, though. fuzztests are supposed to coexist peacefully with normal gtests AFAICT.

It's probably fine excluding the file to work around the problem for now. The fuzztest build isn't important enough (yet) to spend time on. It would be good to add a comment explaining the problem, though.